lua/nvim-tree/log.lua

```diff
@@ -61,6 +61,26 @@ function M.line(typ, fmt, ...)
   end
 end
 
+local inspect_opts = {}
+
+--- @param opts table
+function M.set_inspect_opts(opts)
```

Comment:
Since the inspect options are rarely changed, I chose to add this little function to set them instead of another parameter to the `node` function.